content/post/house-prices-in-king-county.Rmd

```diff
@@ -177,15 +189,23 @@ train.idx = sample(seq_len(tsk$nrow), 0.7 * tsk$nrow)
 test.idx = setdiff(seq_len(tsk$nrow), train.idx)
 ```
 
+We can do the same for our task:
+
+```{r}
+task_train = tsk$clone()$filter(train.idx)
+task_test  = tsk$clone()$filter(test.idx)
```

Comment:
We never use `task_test` ...

Reply:
I thought we might wanna basically eval the "best" learner we found on the held-out data.
But this does not add anything new to the vignette, so I saved the lines.

Are you in favor of adding this?

Reply:
Just add the evaluation in the end then. Otherwise this just leaves the impression that there is something missing.

Reply:
Done.